### DISCUSSION TITLE
Modbus Emulation of Meter

### DISCUSSION THREAD

**Discussion by bubble07** [nathanmarlor/foxess_modbus#486] [Ideas]
Hi Everyone & a happy new year,
I'm in the process of connecting up a Foxess AC coupled inverter (AC1-5.0-E). The installation position needs to be in the garage which is a pain to get a CT clamp to from the main power feed in the house, however I already have a Shelly EM CT clamp working well with home assistant.

My question is; is it possible to control the inverter by telling it what current to charge or discharge at either through the main Modbus interface in the BMS connector or by "spoofing" the Chint / Eastron meter modbus commands through the meter pins in the CT / Meter connector? Thanks!

**Comment by FozzieUK** [comment]
No it wouldn’t be possible, the inverter won’t startup if it can’t balance the loads and if you could write the meter spoof code it wouldn’t be fast enough - this is one of the main reasons that H1 hybrids can’t be connected in parallel as they’re not fast enough to resolve the load balancing.

In all honesty running the CT cable is the simplest solution and the cables can be run 25 meters without issue - another solution if cabling is a non-starter is to use a Lora device to wirelessly connect the inverter to a Chint/Eastron meter, this one has been proven with the Foxess kit https://www.pusr.com/products/serial-rs232-rs485-to-lora-converters-usr-lg206-p.html

**Reply by bubble07** [reply]
Thanks for the reply and link. That's a real shame when I've already got multiple WiFi and modbus CT clamps already installed. I'll have another go at ramming another cat5 cable through the underground conduit 😬.
